### DISCUSSION TITLE
发布准备：下一步计划

### DISCUSSION THREAD

**Discussion by Z-Siqi** [Z-Siqi/Clash-for-Windows_Chinese#151] [Ideas]
或许已经有人注意到了部分地方可能没有汉化到，或是有什么功能因为汉化而导致出问题，如今原版似乎已经不会有任何新发布了，为了后续的使用体验，计划再发布一个Opt版本。因此，如果你发现任何地方有什么问题请向我告知，下个版本将会尽可能修复所有问题

**Comment by Z-fly** [comment]
后续还更新吗

**Reply by Z-Siqi** [reply]
看情况，就算会也不会经常更新。最多就维持在最低的更新频率
